libweb3jsonrpc/test.json

```diff
@@ -1,4 +1,7 @@
 [
+{ "name": "test_getClientInfo", "params": [], "order": [], "returns": ""},
```

Comment:
How is this different to https://github.com/ethereum/wiki/wiki/JSON-RPC#web3_clientversion ? It seems you are just returning the version.

Reply:
in that case. no need to implement a new method.

Reply:
Can you remove it from this PR then?